maths/armstrong_numbers.py

```diff
@@ -53,14 +95,14 @@ def narcissistic_number(n: int) -> bool:
     # check if sum of cube of each digit is equal to number
     return n == sum(temp)
 
-

```

Comment:
Black and flake8 will not like this change.

Reply:
OK